creator-node/src/apiHelpers.ts

```diff
@@ -288,3 +334,23 @@ module.exports.parseCNodeResponse = (respObj, requiredFields = []) => {
     }
   }
 }
+
+module.exports = {
+  handleResponse,
+  handleResponseWithHeartbeat,
+  sendResponse,
+  sendResponseWithHeartbeatTerminator,
+  isValidResponse,
+  errorResponseSocketTimeout,
+  handleApiError,
+  parseCNodeResponse,
+  ErrorBadRequest,
+  ErrorServerError,
+  errorResponseServerError,
+  errorResponseBadRequest,
+  errorResponseNotFound,
+  errorResponse,
+  successResponse,
+  errorResponseForbidden,
+  errorResponseUnauthorized
+}
```

Comment:
was there some error without this? I noticed your other PRs were able to just export without also maintaining a module.exports

Reply:
Same, I've had some break and some not break - I usually keep them both till I'm sure _just_ other typescript files are consuming the file.

Reply:
Let's test it rq

Reply:
haha awesome 🚀